mllib/src/main/scala/org/apache/spark/ml/fpm/PrefixSpan.scala

```diff
@@ -35,7 +37,87 @@ import org.apache.spark.sql.types.{ArrayType, LongType, StructField, StructType}
  */
 @Since("2.4.0")
 @Experimental
-object PrefixSpan {
+final class PrefixSpan(@Since("2.4.0") override val uid: String) extends Params {
+
+  @Since("2.4.0")
+  def this() = this(Identifiable.randomUID("prefixSpan"))
+
+  /**
+   * the minimal support level of the sequential pattern, any pattern that
+   * appears more than (minSupport * size-of-the-dataset) times will be output
+   * (default value: `0.1`).
+   * @group param
+   */
+  @Since("2.4.0")
+  val minSupport = new DoubleParam(this, "minSupport", "the minimal support level of the " +
+    "sequential pattern, any pattern that appears more than (minSupport * size-of-the-dataset) " +
+    "times will be output", ParamValidators.gt(0.0))
+
+  /** @group getParam */
+  @Since("2.4.0")
+  def getMinSupport: Double = $(minSupport)
+
+  /**
+   * Set the minSupport parameter.
+   * Default is 1.0.
```

Comment:
The default is wrong. We don't need doc for the setters and getters. Just leave "@group setParam".